controllers/edgedeployment_controller.go

```diff
@@ -285,3 +342,26 @@ func mergeErrorMessages(errs []error) string {
 	}
 	return message
 }
+
+func splitEdgeDevices(s []managementv1alpha1.EdgeDevice, x uint) [][]managementv1alpha1.EdgeDevice {
```

Comment:
`s` and `x` names do not say anything; please change to something meaningful.

Reply:
please suggest names

Reply:
edgeDevices and splitSize?

Reply:
so splitSize is the size of each slice after the split? this name is confusing. Anyone who reads the function understands immediately what x and s represent. This is why I left it s and x. And if we really want to get philosophical then the type of the slice is meaningless here. Therefore using a name that relates to the type (edgeDevices) is not appropriate in my opinion.
But again, I will change it just to avoid wasting our time in debate.